src/lib/journey/callbacks/recaptcha/recaptcha.svelte

```diff
@@ -6,14 +6,21 @@
   import type { SelfSubmitFunction, StepMetadata } from '$journey/journey.interfaces';
   import type { styleSchema } from '$lib/style.store';
   import type { Maybe } from '$lib/interfaces';
+  import type { StepOptions } from '@forgerock/javascript-sdk/src/auth/interfaces';
 
   export let callback: Maybe<ReCaptchaCallback>;
+  export let journey: Maybe<StepOptions> = null;
   export const selfSubmitFunction: Maybe<SelfSubmitFunction> = null;
   export const stepMetadata: Maybe<StepMetadata> = null;
   export const style: z.infer<typeof styleSchema> = {};
+  // This is the action that the user can pass into the component
+  // if it isn't passed we default to the journey.tree value
+  // journey.tree won't necessarily be set on mount, we have to wait for it
+  // to be passed down so defaulting to an empty string so its falsey
+  export let recaptchaAction = journey?.tree ?? '';
```

Comment:
I seem to be unable to find where `recaptchaAction` is passed into this component. I was expecting to see it as a `journeyEvent.start()` config option and passed through to this component. Am I just missing something?

Also, we'll need to think about where this value custom action value gets stored. Maybe in the journey stack?

Reply:
i must have not committed this...weird.